content/en/blog/2022/collector-builder-sample/index.md

```diff
@@ -0,0 +1,329 @@
+default Collector build, many more components are included in the compiled
+binary (even if they are not being enabled in the Collector’s runtime
+configuration). This is how you can achieve improvements over public images that
+include many excess components, with extra dependencies adding weight and
+security burden. Now, those extra components can’t take up space because they
+aren’t even available in the custom build.
+
+We’ve made this easier to use by including a sample config file in this
+repository which has been pre-filled with a few of the most relevant
+OpenTelemetry components for GCP. However, we encourage you to modify that
+sample file to fit your use case.
+
+For example, the following build file includes only the OTLP receiver and GCP
+exporter, along with a logging exporter:
+
+```
+receivers:
+  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
+    gomod: go.opentelemetry.io/collector v0.57.2
+
+exporters:
+  - import: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter
+    gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.57.2
+  - import: go.opentelemetry.io/collector/exporter/loggingexporter
+    gomod: go.opentelemetry.io/collector v0.57.2
+```
+
+Edit
+[this file](https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample/blob/main/build/local/builder-config.yaml)
+in the repository and run `make build` to automatically generate a local binary,
+or `make docker-build` to compile a container image.
+
+Deploying a custom Collector cuts unneeded dependencies, making these
+trimmed-down builds are faster, lighter, and more secure. In this repository
+we’ve also included the steps required to set these builds up automatically with
+Cloud Build.
+
+## Automate builds for secure, up-to-date images
+
+Along with the steps to build a custom Collector, the new repository also
+includes sample [Cloud Build](https://cloud.google.com/build) configuration to
+support serverless, automated builds for your Collector using Cloud Build
+triggers. By doing so you can benefit from new releases, features, and bug fixes
+in Collector components with minimal delay in your continuous deployment
+pipeline.
+
+When combined with
+[Artifact Registry](https://cloud.google.com/artifact-registry), these builds
+can be pushed as Docker images in your GCP project. This provides portability
+and accessibility of your images, as well as enabling container vulnerability
+scanning in Artifact Registry to ensure the supply-chain safety of your
+Collector deployment.
+
+Serverless builds and vulnerability scanning are valuable, delicate aspects of a
+reliable CI/CD pipeline that are important to get right. Here, we hope to
+abstract away some of the complication around these processes by shipping sample
+configs with this repository.
+
+## Get up and running quickly on GCP
```

Comment:
I think this section could probably be changed to say it's a quick start for setting up otel/collector on GKE.

Reply:
updated